modules/serverless-open-telemetry.adoc

```diff
@@ -96,7 +96,27 @@ spec:
     tracing:
       backend: "zipkin"
       zipkin-endpoint: "http://cluster-collector-collector-headless.tracing-system.svc:9411/api/v2/spans"
-      debug: "true"
+      debug: "true"  <- FIXME do we want consistency here?
```

Comment:
Yes, we want consistency here.

@skonto `debug: true` basically sets `sample-rate` to `1.0` and we don't want this, so debug should be set to false here, right?

Reply:
Yes true. The rationale was back then that with 1.0 rate UX is that the span is going to make it.
So that we don't have questions like where is my span for people who get start with this? It was not meant for production. But I think maybe using a sample-rate: 1.0 is a better UX?